### PR TITLE
css-brochure-form | Nick Hernandez

### PR DESCRIPTION
**Pull request by Nickolas-Hernandez**
Screen Shot 2021-01-20 at 5 45 56 PM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+* {
+  box-sizing: border-box;
+}
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:3]: This was already applied in `reset.css`, it can be removed.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  background-color: rgb(245, 245, 245);
+}
+
+.button-container {
+  width: 100%;
+  text-align: right;
+}
+
+input.submit-button {
+  width: 24%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:51]: You should do this in `px`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Nick, nice work so far! Please see my notes

File: css-brochure-form/styles.css (hunk 2 above)
Comment (Nickolas-Hernandez) [reply css-brochure-form/styles.css]: Screen Shot 2021-01-21 at 1 31 43 PM

Corrections made.

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**